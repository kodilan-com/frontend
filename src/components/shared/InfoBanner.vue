<template>
  <div class="container">
    <div class="sixteen columns">
      <div class="info-banner">
        <div class="info-content">
          <h1>İlanınız burada listelensin!</h1>
          <p>Sizde iş ilanlarınızı birkaç dakika içerisinde ücretsiz olarak yayınlayabilirsiniz.</p>
        </div>
        <router-link to="/ilan-ekle" class="button tag-create-post">
          Ücretsiz İlan Ekle
        </router-link>
        <div class="clearfix" />
      </div>
    </div>
  </div>
</template>
