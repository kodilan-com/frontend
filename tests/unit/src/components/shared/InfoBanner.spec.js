import { shallowMount } from '@vue/test-utils';
import InfoBanner from '@/components/shared/InfoBanner';

describe('InfoBanner.vue', () => {
  it('should match snapshot', () => {
    const wrapper = shallowMount(InfoBanner, {
      stubs: ['router-link'],
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
