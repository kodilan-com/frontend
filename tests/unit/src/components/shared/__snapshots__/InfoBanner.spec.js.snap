// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBanner.vue should match snapshot 1`] = `
<div
  class="container"
>
  <div
    class="sixteen columns"
  >
    <div
      class="info-banner"
    >
      <div
        class="info-content"
      >
        <h3>
          İlanınız burada listelensin!
        </h3>
         
        <p>
          Sizde iş ilanlarınızı birkaç dakika içerisinde ücretsiz olarak yayınlayabilirsiniz.
        </p>
      </div>
       
      <router-link-stub
        class="button tag-create-post"
        to="/ilan-ekle"
      >
        
        Ücretsiz İlan Ekle
      
      </router-link-stub>
       
      <div
        class="clearfix"
      />
    </div>
  </div>
</div>
`;
